Kanban/Controllers/KBNEditProjectViewController.m

```diff
@@ -23,14 +23,16 @@
 #define ALERT_MESSAGE_INVITE_PROMPT_CANCELBUTTONTITLE @"Cancel"
 
 
-@interface KBNEditProjectViewController()
+@interface KBNEditProjectViewController()  <MBProgressHUDDelegate>
 
 @property (weak, nonatomic) IBOutlet UITextField *nameTextField;
 @property (weak, nonatomic) IBOutlet UITextView *descriptionTextView;
-
 @property (strong, nonatomic) IBOutlet UIActivityIndicatorView *activityIndicator;
 @property (strong, nonatomic) IBOutlet UIView *activityIndicatorBackground;
```

Comment:
activityIndicator and activityIndicatorBackground are not used anymore. Remove them from here and from storyboard.